app.py

```diff
@@ -95,8 +61,53 @@ def login():
             'Cache-Control': 'no-cache',
         }
     )
-    root = response.json()['macaroon']
 
+    return response.json()['macaroon']
+
+
+def verify_macaroon(root, discharge):
```

Comment:
I think I'd expect this function to go a bit further and return a boolean of whether the macaroon is verified or not...

Reply:
the problem of doing this way is that I can't really manage the reasons why it's not verified (no access to the package, macaroon expired ...)
Maybe I should call the function `get_verifed_macaroon`

Reply:
Yep fair enough. Nah just leave it. I'm being too fussy.